ecr-login/config/seelog.go

```diff
@@ -15,46 +15,32 @@ package config
 
 import (
 	"fmt"
+	"os"
 	"path/filepath"
 
-	log "github.com/cihub/seelog"
-	homedir "github.com/mitchellh/go-homedir"
+	"github.com/mitchellh/go-homedir"
+	"github.com/sirupsen/logrus"
 )
 
 func SetupLogger() {
-	SetupLoggerWithConfig(loggerConfig())
+	logrusConfig()
 }
 
 func SetupLoggerWithConfig(config string) {
-	logger, err := log.LoggerFromConfigAsString(config)
-	if err == nil {
-		log.ReplaceLogger(logger)
-	} else {
-		log.Error(err)
-	}
 }
 
-func loggerConfig() string {
+func logrusConfig() {
 	logfile, err := homedir.Expand(GetCacheDir() + "/log/ecr-login.log")
 	if err != nil {
 		fmt.Errorf("%v", err)
 		logfile = "/tmp/.ecr/log/ecr-login.log"
 	}
 	// Clean the path to replace with OS-specific separators
 	logfile = filepath.Clean(logfile)
-	config := `
-	<seelog type="asyncloop" minlevel="debug">
-		<outputs formatid="main">
-			<rollingfile filename="` + logfile + `" type="date"
-			 datepattern="2006-01-02-15" archivetype="none" maxrolls="2" />
-			<filter levels="warn,error,critical">
-				<console />
-			</filter>
-		</outputs>
-		<formats>
-			<format id="main" format="%UTCDate(2006-01-02T15:04:05Z07:00) [%LEVEL] %Msg%n" />
-		</formats>
-	</seelog>
-`
-	return config
+	file, err := os.OpenFile(logfile, os.O_RDWR|os.O_CREATE|os.O_APPEND, 0666)
```

Comment:
umask should be respected per `os.OpenFile`'s [godoc](https://godoc.org/os#OpenFile):

> OpenFile is the generalized open call; most users will use Open or Create instead. It opens the named file with specified flag (O_RDONLY etc.) and perm (before umask), if applicable. If successful, methods on the returned File can be used for I/O. If there is an error, it will be of type *PathError.

+1 for 664

Reply:
0666 was used by seelog.  I can change to 0664.

Reply:
Done.